frontend/src/metabase/dashboard/components/Dashboard/stickyParameters.js

```diff
@@ -1,5 +1,7 @@
 import { getMainElement } from "metabase/lib/dom";
 
+export const MAXIMUM_PARAMETERS_FOR_STICKINESS = 6;
+
```

Comment:
Should we really set a threshold for desktop screens too? Six filter widgets can fit nicely on a 13-inch display, not to mention bigger screens. Maybe something like an [`isSmallScreen` check](https://github.com/metabase/metabase/blob/2fe5330a295e240847ac9e6abfcaa48b01782e15/frontend/src/metabase/lib/dom.js#L448) will work better?

Reply:
This can be a potential nice follow-up (since the scope of this PR can be kept to narrow viewport situation).